esp/scm/ws_esdlconfig.ecm

```diff
@@ -214,6 +214,7 @@ ESPrequest [nil_remove] ConfigureESDLBindingLogTransformRequest
     string EsdlBindingId;
     string LogTransformName;
     boolean Overwrite;
+    boolean Escaped(false); //The LogTransform in the Config is not escaped.
```

Comment:
Would this be better as `Encoded`? The value controls whether `encodeXML` is used.

Reply:
Will change